crypto/src/proofs/transparent.rs

```diff
@@ -797,6 +797,9 @@ mod tests {
             amount: 10u64.into(),
             asset_id: asset::REGISTRY.parse_denom("upenumbra").unwrap().id(),
         };
+
+        let balance = -crate::Balance::from(value_to_send);
```

Comment:
I think the test failure is due to mismatching signs, I think we can remove the conversion to Balance and call commit directly on `value_to_send` below (i.e. not `-value_to_send`)

Reply:
I've kept the negated `Balance` following what we talked about (preparing to remove `Value::commit`)